### PR TITLE
feat(crux-ui): separate image and instance config buttons

### PR DESCRIPTION
**Pull request by ngyradi**
Added instance config and log buttons to deploy page.
Added different images for image and instance config buttons.
Tweaked images and icons for consistency and to fix warnings.

### CODE REVIEW

**Review by polaroi8d — APPROVED** [review]
LGTM

File: web/crux-ui/src/components/projects/versions/deployments/deployment-view-list.tsx [hunk 1]
```diff
@@ -57,9 +57,16 @@ const DeploymentViewList = (props: DeploymentViewListProps) => {
     <span suppressHydrationWarning>
       {item.image.createdAt ? utcDateToLocale(item.image.createdAt) : t('common:new')}
     </span>,
-    <Link href={routes.deployment.instanceDetails(state.deployment.id, item.id)} passHref>
-      <DyoIcon src="/settings.svg" alt={t('common:settings')} size="md" />
-    </Link>,
+    <div>
+      <div className="inline-block mr-2">
+        <Link href={routes.project.versions(state.project.id).imageDetails(state.version.id, item.image.id)} passHref>
+          <DyoIcon src="/image_config_icon.svg" alt={t('common:settings')} size="md" />
```
Comment (robot9706) [inline web/crux-ui/src/components/projects/versions/deployments/deployment-view-list.tsx:63]: Both settings button have "Settings" as their tooltip, isn't it a little confusing?

**Review by robot9706 — COMMENTED** [review]
Minor comment, everything else seems to be OK.

**Review by robot9706 — APPROVED** [review]
Edited the DeploymentContainerStatusList component a bit, everything else still looks OK.

**Review by m8vago — APPROVED** [review]

**Pull request merged**